### ISSUE TITLE
docker-compose build

### ISSUE THREAD

**Issue by weteamsteve** [thredded/thredded#853]
When running `docker-compose build` or `docker-compose up`

Step 8/9 RUN bundle install returns
```
You must use Bundler 2 or greater with this lockfile.
ERROR: Service 'web' failed to build: The command '/bin/sh -c bundle install' returned a non-zero code: 20
```

**Comment by tomholford** [comment]
Same here. Took a look at the `Gemfile.lock`, saw this:

```
BUNDLED WITH
   2.1.4
```

So I tried to peg the version in the `Dockerfile`:
```dockerfile
RUN apk add --no-cache \
    ruby ruby-bundler==2.1.4-r0 ruby-bigdecimal ruby-io-console tzdata nodejs bash \
    build-base ruby-dev libc-dev linux-headers gmp-dev openssl-dev libxml2-dev \
    libxslt-dev postgresql-dev
```

Now this seems to be the issue:
```
ERROR: unsatisfiable constraints:
  ruby-bundler-1.12.5-r0:
    breaks: world[ruby-bundler=2.1.4-r0]
```

And this is as far as I got last night :) No idea where this constraint is coming from.
Anyways, here's the full trace for reference:
```
>dc build
WARNING: The LOCAL variable is not set. Defaulting to a blank string.
db uses an image, skipping
Building web
Step 1/10 : FROM alpine:3.4
 ---> b7c5ffe56db7
Step 2/10 : RUN apk add --no-cache     ruby ruby-bundler==2.1.4-r0 ruby-bigdecimal ruby-io-console tzdata nodejs bash     build-base ruby-dev libc-dev linux-headers gmp-dev openssl-dev libxml2-dev     libxslt-dev postgresql-dev
 ---> Running in 3f6a28c4dfd8
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  ruby-bundler-1.12.5-r0:
    breaks: world[ruby-bundler=2.1.4-r0]
ERROR: Service 'web' failed to build: The command '/bin/sh -c apk add --no-cache     ruby ruby-bundler==2.1.4-r0 ruby-bigdecimal ruby-io-console tzdata nodejs bash     build-base ruby-dev libc-dev linux-headers gmp-dev openssl-dev libxml2-dev     libxslt-dev postgresql-dev' returned a non-zero code: 1
```

**Comment by glebm** [comment]
In the log you have:

```
Step 1/10 : FROM alpine:3.4
```

This is an old version of alpine and is not what is specified in the Docker file. The log should say

```
Step 1/11 : FROM alpine:edge
```

**Comment by glebm** [comment]
There are other issues with the current docker setup. I'm looking into them

**Comment by tomholford** [comment]
Thanks @glebm for the quick fix :muscle:

**Comment by glebm** [comment]
I've updated thredded_create_app as well (released v0.2.2)
